src/EditorFeatures/CSharpTest/EditAndContinue/TopLevelEditingTests.cs

```diff
@@ -7183,6 +7701,54 @@ class C
             edits.VerifySemanticDiagnostics();
         }
 
+        [Fact, WorkItem(17681, "https://github.com/dotnet/roslyn/issues/17681")]
+        public void Event_ExpressionBodyToBlockBody()
+        {
+            string src1 = @"
+using System;
+public class C
+{
+    event Action E { add => F(); remove => F(); }
+}
+";
+            string src2 = @"
+using System;
+public class C
+{
+   event Action E { add { F(); } remove => F(); }
```

Comment:
ok and below

---
In reply to: [128833458](https://github.com/dotnet/roslyn/pull/20931#discussion_r128833458) [](ancestors = 128833458)